src/test/datascience/debugger.functional.test.tsx

```diff
@@ -155,8 +158,9 @@ suite('DataScience Debugger tests', () => {
         }
 
         // Start the jupyter server
+        traceInfoIf(!!process.env.VSC_CI_ENABLE_TOO_MUCH_LOGGING, 'Getting interactive window');
         const history = await getOrCreateInteractiveWindow(ioc);
-
+        traceInfoIf(!!process.env.VSC_CI_ENABLE_TOO_MUCH_LOGGING, 'Got interactive window');
```

Comment:
Added some logging to find out what part is hanging when the variable tests timeout.